yarn.lock

```diff
@@ -16130,7 +16130,7 @@ express-prom-bundle@6.6.0:
     on-finished "^2.3.0"
     url-value-parser "^2.0.0"
 
-express-promise-router@^4.1.0, express-promise-router@^4.1.1:
+express-promise-router@4.1.1, express-promise-router@^4.1.0, express-promise-router@^4.1.1:
```

Comment:
4.1.1 is defined twice; is this ok?

Reply:
Oh one is for 4.11 and the other is for ^4.1.1, so I don't think it should be an issue?